helix-query.yaml

```diff
@@ -32,9 +32,13 @@ indices:
       subcategory:
         select: head > meta[name="subcategory"]
         value: attribute(el, "content")
-      visibility:
-        select: head > meta[name="visibility"]
+      category:
+        select: head > meta[name="category"]
         value: attribute(el, "content")
       category:
         select: head > meta[name="category"]
         value: attribute(el, "content")
+      keywords:
+        select: head > meta[name="keywords"]
+        value: attribute(el, "content")
```

Comment:
it should be values if we will be having multiple keywords.

Reply:
Its the same name as on original site. I want to maintain the consistency

Reply:
also, this PR removes visibility tag

Reply:
Adding more context to be clearer,

if we want to store more than 1 value to a property, we have to use **values**: attribute(el, "content") instead of **value**: attribute(el, "content")